pkg/db/v1beta1/mysql/init.go

```diff
@@ -18,22 +18,34 @@ package mysql
 
 import (
 	"fmt"
-
+	"github.com/kubeflow/katib/pkg/db/v1beta1/common"
+	"github.com/kubeflow/katib/pkg/util/v1beta1/env"
 	"k8s.io/klog"
 )
```

Comment:
```suggestion
	"fmt"

	"k8s.io/klog"

	"github.com/kubeflow/katib/pkg/db/v1beta1/common"
	"github.com/kubeflow/katib/pkg/util/v1beta1/env"
```

Let's grouping dependencies.